### PR TITLE
#19 feature: Track order status 

### LINKED ISSUE #36: Deprecated packages
When you run a fresh install look at the deprecation warnings to see the whole list and the relevant messages.
- uuid 
- codeclimate-test-reporter
- tslint
- etc

### PR DESCRIPTION
**Pull request by yobuir**
#### What does this PR do?
A buyer should be able to track their order status
#### Description of Task to be completed?
A buyer should be able to know the status of his/her order after sign in 
#### How should this be manually tested?
```
git clone https://github.com/atlp-rwanda/ecommerce-app-predators-fe.git
cd ecommerce-app-predators-fe
git checkout track-order-status-19
npm i
npm run dev
```
#### What are the relevant pivotal tracker/Trello stories?
Trello [Link](https://trello.com/c/ewShIM4D/19-a-buyer-should-be-able-to-track-their-order-status)

#### Screenshots (if appropriate)

### CODE REVIEW

**Review by ALVINdimpos — APPROVED** [review]

**Review by nili-alu — APPROVED** [review]

File: bash.exe.stackdump [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Stack trace:
```
Comment (chanceDira) [inline bash.exe.stackdump]: Can you clarify the use of this file?

File: src/pages/buyer/Orders.tsx [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+ 
+interface Product {
+  name: string | number | null | undefined;
+  quantity: number | null | undefined;
+  subtotal: number | null | undefined;
+}
+
+ 
+interface OrdersData {
+  data: any | null | undefined;
```
Comment (chanceDira) [inline src/pages/buyer/Orders.tsx]: data inside the OrdersData interface can have first interface then other options like null and undefined

**Review by chanceDira — CHANGES_REQUESTED** [review]
Good work, just some requested changes below

**Review by vanessa-abewe — APPROVED** [review]

**Review by chanceDira — APPROVED** [review]
Looking good

**Pull request merged**